lib/masamune/schema/column.rb

```diff
@@ -287,7 +287,9 @@ def ruby_value(value, recursive = true)
         when Hash
           value
         when String
-          JSON.parse(value)
+          JSON.parse(value).tap do |new_value|
+            raise unless new_value.is_a?(Hash) || new_value.is_a?(Array)
+          end
```

Comment:
Guessing its the "[Now complies to newest JSON RFC 7159.](https://github.com/flori/json/blob/master/CHANGES.md#2015-09-11-200)" change in JSON library